### PR TITLE
update powerby page 

### PR DESCRIPTION
**Pull request by helenlly**
1. re-org
2.add new tech docs and wp

### CODE REVIEW

**Comment by hkvision** [comment]
LGTM

File: docs/docs/powered-by.md [hunk 1]
```diff
@@ -1,15 +1,40 @@
 ## Powered By
 ---
-* __Microsoft Azure__: 
+* __1. Alibaba__
+  <br>[Deploy Analytics Zoo in Aliyun EMR](https://partners-intl.aliyun.com/help/doc-detail/93155.htm )
+* __2. Baosight__
+  <br>[LSTM-Based Time Series Anomaly Detection Using Analytics Zoo for Apache Spark* and BigDL at Baosight](https://software.intel.com/en-us/articles/lstm-based-time-series-anomaly-detection-using-analytics-zoo-for-apache-spark-and-bigdl)
+* __3. Cern__
```
Comment (jason-dai) [inline docs/docs/powered-by.md]: CERN

File: docs/docs/powered-by.md [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
+ <br>[Topology classification at CERN’s Large Hadron Collider using Analytics Zoo](https://db-blog.web.cern.ch/blog/luca-canali/machine-learning-pipelines-high-energy-physics-using-apache-spark-bigdl)
+ * __4. China Telecom__
+ <br>[Face Recognition Application and Practice Based on Intel Analytics Zoo: Part 1](https://mp.weixin.qq.com/s/FEiXoTDi-yy04PJ2Mlfl4A)(Chinese Version)
+ <br>[Face Recognition Application and Practice Based on Intel Analytics Zoo: Part 2](https://mp.weixin.qq.com/s/VIyWRORTAVAAsC4v6Fi0xw)(Chinese Version)
+* __5. Cray__ 
+<br>[A deep learning approach for precipitation nowcasting with RNN using Analytics Zoo in Cray](https://conferences.oreilly.com/strata/strata-ny-2018/public/schedule/detail/69413)
+* __6. Dell EMC__
+<br>[[White Paper] AI-assisted Radiology Using Distributed Deep
```
Comment (jason-dai) [inline docs/docs/powered-by.md]: remove whitepaper

File: docs/docs/powered-by.md [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
+* __13. Telefonica__
+ <br>[Running Analytics Zoo jobs on Telefónica Open Cloud’s MRS Service](https://medium.com/@fernando.delaiglesia/running-analytics-zoo-jobs-on-telef%C3%B3nica-open-clouds-mrs-service-2e64bc823c50)
+* __14. Tencent__
+<br>[Enhance Tencent's TUSI Identity Practice with Intel Analytics Zoo](https://mp.weixin.qq.com/s?__biz=MzAwNzc5NzM5Mw==&mid=2651030944&idx=1&sn=d6e06c6e14a7355971953a501689b232&chksm=808f8a5eb7f80348fc8e88c4c9e415341bf43ef6bdf3fd4f3001da89e2c9ba7fa2ed5deeb09a&mpshare=1&scene=1&srcid=0412WxM3eWdsLLoO2TYJGWbS&pass_ticket=E6l%2FfOZNKjhr05lsU7inAVCi7mAy5LFEehvEJOS2ZGdHg6%2FH%2BeBQisHA9sfXDOoy#rd)(Chinese Version)
+* __15. World Bank__
+<br>[Deep Learning and Artificial Intelligence Help World Bank Team Create Image-Recognition Models for Crowdsourced Photos](https://itpeernetwork.intel.com/artificial-intelligence-world-bank-image-recognition/)
+* __16. Yunda__
+<br>[[White Paper] Intelligent transformation brings "quality change" to the express delivery industry](https://www.intel.cn/content/www/cn/zh/analytics/artificial-intelligence/yunda-brings-quality-change-to-the-express-delivery-industry.html)(Chinese Version)
```
Comment (jason-dai) [inline docs/docs/powered-by.md]: remove whitepaper

File: docs/docs/powered-by.md [hunk 4]
```diff
@@ -1,15 +1,40 @@
...
-* __Alibaba__: [Deploy Analytics Zoo in Aliyun EMR](https://yq.aliyun.com/articles/638782)
-* __Baosight__: [LSTM-Based Time Series Anomaly Detection Using Analytics Zoo for Apache Spark* and BigDL at Baosight](https://software.intel.com/en-us/articles/lstm-based-time-series-anomaly-detection-using-analytics-zoo-for-apache-spark-and-bigdl)
-* __Midea__: [Industrial Inspection Platform in Midea* and KUKA*: Using Distributed TensorFlow* on Analytics Zoo](https://software.intel.com/en-us/articles/industrial-inspection-platform-in-midea-and-kuka-using-distributed-tensorflow-on-analytics) 
+<br>[Use Analytics Zoo to Inject AI Into Customer Service Platforms on Microsoft Azure: Part 2](https://www.infoq.com/articles/analytics-zoo-qa-module/?from=timeline&isappinstalled=0)
+* __10. Midea__
+<br>[[White Paper] Ability to add "eyes" and "brains" to smart manufacturing](https://www.intel.cn/content/www/cn/zh/analytics/artificial-intelligence/midea-case-study.html)(Chinese Version)
```
Comment (jason-dai) [inline docs/docs/powered-by.md]: remove whitepaper

**Comment by jason-dai** [comment]
Also add links for talks

File: docs/docs/powered-by.md (hunk 1 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

File: docs/docs/powered-by.md (hunk 2 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

File: docs/docs/powered-by.md (hunk 3 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

File: docs/docs/powered-by.md (hunk 4 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

**Comment by helenlly** [comment]
> Also add links for talks

added

File: docs/docs/powered-by.md [hunk 5]
```diff
@@ -1,15 +1,45 @@
...
+<br>[Job recommendations leveraging deep learning using Analytics Zoo on Apache Spark and BigDL](https://conferences.oreilly.com/strata/strata-ny-2018/public/schedule/detail/69113)
+* __13. Telefonica__
+ <br>[Running Analytics Zoo jobs on Telefónica Open Cloud’s MRS Service](https://medium.com/@fernando.delaiglesia/running-analytics-zoo-jobs-on-telef%C3%B3nica-open-clouds-mrs-service-2e64bc823c50)
+* __14. Tencent__
+<br>[Enhance Tencent's TUSI Identity Practice with Intel Analytics Zoo](https://mp.weixin.qq.com/s?__biz=MzAwNzc5NzM5Mw==&mid=2651030944&idx=1&sn=d6e06c6e14a7355971953a501689b232&chksm=808f8a5eb7f80348fc8e88c4c9e415341bf43ef6bdf3fd4f3001da89e2c9ba7fa2ed5deeb09a&mpshare=1&scene=1&srcid=0412WxM3eWdsLLoO2TYJGWbS&pass_ticket=E6l%2FfOZNKjhr05lsU7inAVCi7mAy5LFEehvEJOS2ZGdHg6%2FH%2BeBQisHA9sfXDOoy#rd)(Chinese Version)
+<br>[Data science service built on top of Analytics Zoo in Tencent Cloud Sparkling product](https://ai.oreilly.com.cn/ai-cn/public/schedule/detail/75442)
```
Comment (jason-dai) [inline docs/docs/powered-by.md]: remove this one

**Comment by jason-dai** [comment]
> > Also add links for talks
> 
> added

Ho about other talks such as World Bank, etc?

File: docs/docs/powered-by.md [hunk 6]
```diff
@@ -1,15 +1,41 @@
 ## Powered By
```
Comment (hkvision) [inline docs/docs/powered-by.md]: Remove this line. Redundant, see here https://analytics-zoo.github.io/master/#powered-by/

File: docs/docs/powered-by.md [hunk 7]
```diff
@@ -1,15 +1,41 @@
...
-* __Midea__: [Industrial Inspection Platform in Midea* and KUKA*: Using Distributed TensorFlow* on Analytics Zoo](https://software.intel.com/en-us/articles/industrial-inspection-platform-in-midea-and-kuka-using-distributed-tensorflow-on-analytics) 
-
+<br>[Use Analytics Zoo to Inject AI Into Customer Service Platforms on Microsoft Azure: Part 2](https://www.infoq.com/articles/analytics-zoo-qa-module/?from=timeline&isappinstalled=0)
+* __10. Midea__
+<br>[Ability to add "eyes" and "brains" to smart manufacturing](https://www.intel.cn/content/www/cn/zh/analytics/artificial-intelligence/midea-case-study.html)(Chinese Version)
+<br>[Industrial Inspection Platform in Midea* and KUKA*: Using Distributed TensorFlow* on Analytics Zoo](https://software.intel.com/en-us/articles/industrial-inspection-platform-in-midea-and-kuka-using-distributed-tensorflow-on-analytics) 
+* __11. Office Depot__
+<br>[Office Depot product recommender using Analytics Zoo on AWS](https://conferences.oreilly.com/strata/strata-ca/public/schedule/detail/73079)
```
Comment (hkvision) [inline docs/docs/powered-by.md]: missing the blog for office depot?

File: docs/docs/powered-by.md (hunk 6 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

File: docs/docs/powered-by.md (hunk 7 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: added

File: docs/docs/powered-by.md (hunk 5 above)
Comment (helenlly) [reply docs/docs/powered-by.md]: done

**Comment by helenlly** [comment]
> > > Also add links for talks
> > 
> > 
> > added
> 
> Ho about other talks such as World Bank, etc?

Dell/Office Depot/WorldBank/Cray/Taloo talks are added

**Pull request merged**